pkg/verify/testdata/manifests/loads-valid.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: release-verification
+  namespace: configmap.NamespaceLabelConfigMap
```

Comment:
Shouldn't these be literal `openshift-config-managed`, a not the Go variable?  Although the docs are pretty clear that only the annotation matters, so maybe we want `namespace: only-the-annotation-matters` ;).

Reply:
They should. Fixed.